[minor] remove usage of Kernel::$rootDir from docs

jzawadzki opened the pull request:
As per https://github.com/symfony/symfony/pull/28810 $rootDir has been removed in 5.0 - use getProjectDir in docs instead - small change to be consistent

erlangparasu committed:
uppercase

OskarStark committed:
minor #12707 uppercase (erlangp)

This PR was submitted for the 5.0 branch but it was merged into the 4.3 branch instead (closes #12707).

Discussion
----------

uppercase

Commits
-------

6ba282e22 uppercase

jzawadzki committed:
[minor] remove usage of Kernel::$rootDir from docs

jzawadzki committed:
Changed names of Intl classes in form types docs

javiereguiluz commented on reference/configuration/kernel.rst:
```diff
@@ -87,7 +87,7 @@ method to return the right project directory::
 ~~~~~~~~~~~~~~~
 
-**type**: ``string`` **default**: ``$this->rootDir/cache/$this->environment``
+**type**: ``string`` **default**: ``$this->getProjectDir()/var/cache/'$this->environment``
```
We should remove the extra ``'`` before `$this->environment`

javiereguiluz reviewed (approved):
Nice catch! Thanks.

OskarStark committed:
minor #12720 Changed names of Intl classes in form types docs (jzawadzki)

This PR was submitted for the 5.0 branch but it was merged into the 4.3 branch instead (closes #12720).

Discussion
----------

Changed names of Intl classes in form types docs

As per https://github.com/symfony/symfony/pull/28846 we changed Intl classes to be more specific. Looks like some docs left with old naming.

Commits
-------

f5f64e1b3 Changed names of Intl classes in form types docs

wouterj reviewed (approved):
Great catch!

As this method is already available in 4.x, let's merge this into 4.3. You don't have to do anything for this (our merge tool can take care of it).

javiereguiluz committed:
Merge remote-tracking branch 'upstream/4.3' into 4.3

* upstream/4.3:
  Changed names of Intl classes in form types docs
  uppercase

javiereguiluz commented:
Thanks! (merged in 4.3)
